src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <section id="wrapper" class="section is-flex is-justify-content-center is-align-items-center">
+    <div class="container">
+      <div class="columns is-centered">
+        <div class="column is-half" id="errorContainer">
+          
+          <div class="columns">
+            <div class="column is-three-fifths">
+              <div class="field has-addons">
+                <div class="control is-expanded">
+                  <input class="input inputCity" type="search" id="address-input" placeholder="City Name">
+                </div>
+              </div>
+            </div>
+            <div class="column">
+              <div class="control">
+                <a class="btnLocation button is-info is-fullwidth">
+                  Get my Location
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="columns has-text-centered">
+            <div class="column is-three-fifths has-addons-left">
+              <div class="has-text-light">
+                <p id="todayDate">Data</p>
+              </div>
+              <div>
+                <p id="btnTemp" class="temperature temp-celsius has-text-warning is-size-1">?°C</p>
+              </div>
+              <div>
+                <p class="location has-text-light is-size-5">location, country</p>
+              </div>
+            </div>
+            <div class="column">
+              <div class="icon-div">
+                <div class="weather-icon"><img src="icons/unknown.png" alt=""></div>
```

Comment:
- [ ] There is an console error coming from the image source. Let's fix the used image path
![Screenshot from 2020-12-15 00-20-03](https://user-images.githubusercontent.com/4246637/102142588-56959b00-3e6b-11eb-80ec-7b3c15585fb5.png)